lib/utils/errors.mjs

```diff
@@ -41,3 +41,25 @@ export class NoFilesFoundError extends Error {
 		this.message = `No files matching the pattern "${pattern}" were found.`;
 	}
 }
+
+export class SuppressionFileNotFoundError extends Error {
+	name = 'SuppressionFileNotFoundError';
+
```

Comment:
```suggestion

	code = EXIT_CODE_INVALID_USAGE;

```

I think this code would be on point.

> The command was used incorrectly, e.g., with the wrong number of arguments, a bad flag, a bad syntax in a parameter, or whatever.

Reply:
Thanks, fixed.
→ [Change code for not found error to EXIT_CODE_INVALID_USAGE](https://github.com/stylelint/stylelint/pull/8564/commits/ae59af05adbe92ab90742c1ed5d3b89c04012c40)